server/emails/templates/BaseEmail.tsx

```diff
@@ -130,6 +165,7 @@ export default abstract class BaseEmail<
     if (notification) {
       try {
         notification.emailedAt = new Date();
+        notification.messageId = messageId ?? null;
```

Comment:
Return `messageId ?? null` from `sendMail` perhaps?

Reply:
Idea was to ensure the interfaces don't return `null` in the codebase. Thoughts?